modules/core/src/main/scala/org/tessellation/l0/domain/snapshot/GlobalSnapshot.scala

```diff
@@ -8,12 +8,15 @@ import org.tessellation.schema.height.Height
 import org.tessellation.schema.peer.PeerId
 import org.tessellation.security.signature.Signed
 
-import io.estatico.newtype.ops._
+import eu.timepit.refined.types.numeric.NonNegLong
+
+case class GlobalSnapshotOrdinal(value: NonNegLong)
```

Comment:
could it be a newtype?

Reply:
Oh yes, I forgot to make it a newtype. I just need to move that to package object